src/components/ActivityStream.js

```diff
@@ -167,6 +180,22 @@ const ActivityStreamList = ({ activities, lastSeenAt, lastUpdatedAt }) => {
             return acc;
           }
 
+          const info =
+            data.from === safeAddress
+              ? { prefix: '-', type: 'SENT' }
+              : { prefix: '+', type: 'RECEIVED' };
+
```

Comment:
Both `if` statements could be in the same one as both are used for returning the same value

Reply:
With an OR || between statements on line 189-190 and line 195, improve readability :)